app/helpers/bill_parser.rb

```diff
@@ -0,0 +1,40 @@
+	module BillConverter
```

Comment:
The formatting of this file is a bit crazy. Formatting is very important!

Do you need a module here or can you just wrap this file in a `helpers do…end` block?